sentences/ro/_common.yaml

```diff
@@ -47,10 +47,18 @@ lists:
       - in: "f"
         out: "fahrenheit"
 expansion_rules:
-  area: "[în | din] {area}"
-  brightness: "{brightness:brightness_pct} [la sută]"
+  area: "{area}"
+  brightness: "{brightness:brightness_pct} [(la suta | la sută | %)]"
   temperature: "{temperature} [grade] [{temperature_unit}]"
   name: "({name}[ul] | {name})"
+  change: "(schimba | schimbă | modifica | modifică | seteaza | setează)"
+  from: "(din | in | în | pentru)"
+  start: "(start | pornește | porneste | deschide | aprinde | activează | activeaza)"
+  stop: "(stop | oprește | opreste | închide | inchide | stinge | dezactivează | dezactiveaza)"
+  percent: "(la suta | la sută)"
+  light: "(luminile | lumina | becul | becurile)"
```

Comment:
Expansion rules that do not wrap a slot (ie `{name}`) should be in native language, not in English.